tools/gce_setup/grpc_docker.sh

```diff
@@ -1274,8 +1274,7 @@ grpc_cloud_prod_gen_go_cmd() {
 #   cmd=$($grpc_gen_test_cmd $flags)
 grpc_interop_gen_java_cmd() {
     local cmd_prefix="sudo docker run grpc/java";
-    local test_script="/var/local/git/grpc-java/run-test-client.sh";
-    local test_script+=" --use_test_ca=true --use_tls=true"
+    local test_script="/var/local/git/grpc-java/run-test-client.sh --use_test_ca=true --use_tls=true";
```

Comment:
I think it was the equivalent of word wrapping. grpc_cloud_prod_gen_go_cmd does the same thing. grpc_cloud_prod_auth_compute_engine_creds_gen_ruby_cmd too. grpc_interop_gen_php_cmd too. Which ones are you seeing that don't do it?